src/stk/calculators/energy/energy_calculators.py

```diff
@@ -128,29 +130,12 @@ def inner(self, mol):
     return inner
 
 
-class EnergyCalculator:
+class EnergyCalculator(Calculator):
     """
     Calculates the energy of molecules.
 
     """
 
-    def __init__(self, use_cache=False):
-        """
-        Initialize a :class:`EnergyCalculator` instance.
-
-        Parameters
-        ----------
-        use_cache : :class:`bool`, optional
-            If ``True`` :meth:`get_energy` will not run twice on the
-            same molecule, but will instead return the previously
-            calculated value.
-
-        """
-
-        # Maps molecules to previously calculated energy values.
-        self._cache = {}
-        self._use_cache = use_cache
-
     def __init_subclass__(cls, **kwargs):
         cls.get_energy = _add_cache_use(cls.get_energy)
```

Comment:
A comment would be quite useful here explaining that any subclasses' `get_energy` function will be decorated with cache use

Reply:
This is getting addressed in a future PR, in that the entire line of code is being removed.